trigger/host.go

```diff
@@ -26,6 +28,10 @@ const (
 	expiringHostEmailBody            = `Your {{.Distro}} host '{{.Name}}' will be terminated at {{.ExpirationTime}}. Visit the <a href={{.URL}}>spawnhost page</a> to extend its lifetime.`
 	expiringHostSlackBody            = `Your {{.Distro}} host '{{.Name}}' will be terminated at {{.ExpirationTime}}. Visit the <{{.URL}}|spawnhost page> to extend its lifetime.`
 	expiringHostSlackAttachmentTitle = "Spawn Host Page"
+
+	idleHostEmailSubject     = `{{.Distro}} idle host notice`
+	idleStoppedHostEmailBody = `Your stopped {{.Distro}} host '{{.Name}}' has been idle since {{.LastCommunicationTime}}. 
+In order to be responsible about resource consumption, please consider stopping or removing from the <a href={{.URL}}>spawnhost page</a> if the host is no longer in use.`
```

Comment:
I'd suggest mentioning that even paused instances cost money, since users might assume a paused instance is free.